README.md

```diff
@@ -142,57 +171,54 @@ For more extensive examples see [EXAMPLES.md](./EXAMPLES.md).
 - [getAccessToken](https://auth0.github.io/nextjs-auth0/modules/session_get_access_token.html)
 - [initAuth0](https://auth0.github.io/nextjs-auth0/modules/instance.html)
 
-**Client Side methods/components**:
+**Client-side methods/components**:
 
 - [UserProvider](https://auth0.github.io/nextjs-auth0/modules/frontend_use_user.html#userprovider)
 - [useUser](https://auth0.github.io/nextjs-auth0/modules/frontend_use_user.html)
 - [withPageAuthRequired](https://auth0.github.io/nextjs-auth0/modules/frontend_with_page_auth_required.html)
 
-Generated [API Docs](https://auth0.github.io/nextjs-auth0/)
+Visit the auto-generated [API Docs](https://auth0.github.io/nextjs-auth0/) for more details.
 
 ### Cookies and Security
 
-All cookies will be set as `HttpOnly, SameSite=Lax` and will be forced to HTTPS (`Secure`) if the application's `AUTH0_BASE_URL` is `https`.
+If the application's `AUTH0_BASE_URL` is `https`, the SDK will set all cookies to `HttpOnly, SameSite=Lax` with the `Secure` attribute to force HTTPS.
```

Comment:
The SDK will set all cookies `HttpOnly, SameSite=Lax` regardless of the protocol of the base url